modules/standard/Spawn.chpl

```diff
@@ -128,6 +131,17 @@ module Spawn {
 
   /* 
      This record represents a subprocess.
+
+     Note that the subprocess will not be waited for if this record
+     goes out of scope. Memory used by the subprocess record itself
+     to implement pipes will be freed if the record goes out of scope,
+     however.
+
+     Generally, it is important to call :proc:`subprocess.wait` to wait for the
+     process to complete. If the parent process is using pipes to communicate
+     with the subprocess, the parent process may call :proc:`subprocess.close`
+     in order to close the pipes and free any buffers.
```

Comment:
Well... technically subprocess.close isn't called if it goes out of scope - instead the individual channels are released/closed individually, because that is what the default record destructor does. The paragraph just above says that the channels/pipes will be closed/freed....  I could clarify it here with an additional sentence I suppose.